doc/content/modules/user-manual/pages/release-notes/2025.8.0.adoc

```diff
@@ -58,6 +58,46 @@ Type T3 {
 
 The `FeatureChain` _t1Feature.t2Feature_ is now typed with _T3_.
 
+- Fix a name resolution problem during textual import for elements used in a `FlowConnectionAsUsage`.
```

Comment:
during textual import for elements used => during textual import of elements used